lib/config/presets/internal/config.ts

```diff
@@ -4,6 +4,7 @@ export const presets: Record<string, Preset> = {
   base: {
     description: 'Default base configuration for all languages',
     extends: [
+      ':dependencyDashboard',
```

Comment:
I'm confused as to why this shows up as an addition. Pretty sure that it's already in v26